AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -217,6 +220,21 @@ - (void)fetchData
       _player = [[AVPlayer alloc] initWithPlayerItem:_currentItem];
       _player.muted = _muted;
     }
+    __weak __typeof(self) weakSelf = self;
+    _timeObserver = [_player addPeriodicTimeObserverForInterval:CMTimeMake(1, 1) queue:NULL usingBlock:^(CMTime time){
```

Comment:
I would prefer to move them to delegate like @ejensen has mentioned, because this way we have the flexibility to register any observer as we desire.